### DISCUSSION TITLE
Inline mode issue

### DISCUSSION THREAD

**Discussion by igorpanin** [n3d1117/chatgpt-telegram-bot#581] [Q&A]
ERROR - An error occurred while generating the result card for inline query _init_
() got an unexpected keyword argument 'thumb_url'

![image](https://github.com/n3d1117/chatgpt-telegram-bot/assets/125222668/5594ab67-0d30-4972-8cdf-d1b134c61fd0)

Any resolution? Thank you

**Comment by OKyryllov** [comment]
Commited a fix for you https://github.com/n3d1117/chatgpt-telegram-bot/pull/582
